### PR TITLE
Adding project BlockAI/ BAI

### PR DESCRIPTION
**Pull request by christopheSeeka**
Adding BAI / BlockAI to ecosystem.json

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/data/ecosystem.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[
+o[
```
Comment (wbnns) [inline apps/web/src/data/ecosystem.json:1]: This change should not be part of the PR

**Review by wbnns — CHANGES_REQUESTED** [review]
@christopheSeeka 

Hello, thanks for the new PR. There are two changes that would break the ecosystem page that seem to have been brought in with the code change. I've marked them in-line. There was also a small conjugation issue. Please fix. The rest of your PR LGTM. 👍

File: apps/web/src/data/ecosystem.json [hunk 2]
```diff
@@ -3606,5 +3606,13 @@
     "imageUrl": "/images/partners/tribally.png",
     "category": "consumer",
     "subcategory": "gaming"
+  },
+  {
+    "name": "BlockAI",
+    "category": "infra",
+    "subcategory": "ai",
+    "description": "BlockAI combines blockchain and AI to offer decentralized access to innovative tools like Text and image generation, video summarization, games etc.",
```
Comment (wbnns) [inline apps/web/src/data/ecosystem.json:3614]: Please add comma between games and etc.

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-12-28 12:29:00 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wbnns — CHANGES_REQUESTED** [review]
Restarting Heimdall‎

**Comment by christopheSeeka** [comment]
Updated, let me know if it's ok.

**Comment by wbnns** [comment]
@christopheSeeka Hi, this still needs to be rebased. 🙏

**Comment by christopheSeeka** [comment]
Ok will do, just to be sure is itmy rebase that arewrong or it isbecause thereisnew commitsin between approval time? 😇

**Pull request closed without merging**